style.css

```
@@ -1,10 +1,158 @@
-body{
-  background-color:#ECE3CE;
-  font-size:16px;
-  font-family: 'Figtree', sans-serif;
+body {
+    background-color: #ECE3CE;
+    font-size: 16px;
+    font-family: 'Figtree', sans-serif;
+    color: #333333;
+    margin: 0;
 }
 
-h1{
-    color: #4F6F52;
+header {
+    max-width: 100%;
+    background-color: #739072;
+    box-sizing: border-box;
+    padding: 20px;
+}
+
+nav {
+    max-width: 1200px;
+    margin: 0 auto;
+}
+
+.logo {
+    width: 120px;
+}
+
+main {
+    max-width: 1200px;
+    margin: 0 auto;
+    padding: 0 20px;
+}
+
+.hero-container {
+    max-width: 100%;
+    height: 60vh;
+    background: url(images/falloftheusherhouse.jpg) no-repeat center;
+    background-size: cover;
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: flex-end;
+    margin: 20px 0;
+
+}
+
+.gradient-layer {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    background: rgb(25, 34, 25);
+    background: -moz-linear-gradient(0deg, rgba(25, 34, 25, 1) 0%, rgba(58, 77, 57, 0) 63%);
+    background: -webkit-linear-gradient(0deg, rgba(25, 34, 25, 1) 0%, rgba(58, 77, 57, 0) 63%);
+    background: linear-gradient(0deg, rgba(25, 34, 25, 1) 0%, rgba(58, 77, 57, 0) 63%);
+}
+
+.hero-content {
+    z-index: 1;
+    flex: 1 1 auto;
+    text-align: center;
+    margin: 20px;
+}
+
+.hero-content p, h1{
+    margin: 0;
+}
+
+.hero-content p {
+    color: #fff;
+    text-transform: uppercase;
+    letter-spacing: 5px;
+}
+
+.hero-content h1 {
+    color: #739072;
     font-size: 2.6em;
+}
+
+h2 {
+   
+    font-size: 1.8em;
+}
+
+p {
+    line-height: 150%;
+}
+
+.message-container{
+    background-color: #fff;
+    border-radius: 8px;
+    width: 400px;
+    box-sizing: border-box;
+    padding: 20px;
+    margin: 30px auto;
+    max-width: 100%;
+    box-shadow: rgba(207, 200, 183, 0.6) 0px 20px 25px -5px, rgba(99, 93, 80, 0.1) 0px 10px 10px -5px;
+}
+
+.message-container p{
+font-weight: 600;
+font-size: 18px;
+text-align: center;
+color: #739072;
+}
+
+.message-container input, textarea{

```

```suggestion
.message-container input, textarea {

```

(satöbbi, ilyenből van még néhány)